pluma/pluma-window.c

```diff
@@ -1281,2841 +1274,2814 @@ recent_manager_changed (GtkRecentManager *manager,
+    gtk_widget_show (main_box);
+
+    /* Add menu bar and toolbar bar */
+    create_menu_bar_and_toolbar (window, main_box);
+
+    /* Add status bar */
+    create_statusbar (window, main_box);
+
+    /* Add the main area */
+    pluma_debug_message (DEBUG_WINDOW, "Add main area");
+    window->priv->hpaned = gtk_paned_new (GTK_ORIENTATION_HORIZONTAL);
+    gtk_box_pack_start (GTK_BOX (main_box),
+                        window->priv->hpaned,
+                        TRUE,
+                        TRUE,
+                        0);
+
+    window->priv->vpaned = gtk_paned_new (GTK_ORIENTATION_VERTICAL);
+      gtk_paned_pack2 (GTK_PANED (window->priv->hpaned),
+                       window->priv->vpaned,
+                       TRUE,
+                       FALSE);
+
+    pluma_debug_message (DEBUG_WINDOW, "Create pluma notebook");
+    window->priv->notebook = pluma_notebook_new ();
+    add_notebook (window, window->priv->notebook);
+
+    /* side and bottom panels */
+      create_side_panel (window);
+    create_bottom_panel (window);
+
+    /* panes' state must be restored after panels have been mapped,
+     * since the bottom pane position depends on the size of the vpaned. */
+    window->priv->side_panel_size = g_settings_get_int (window->priv->editor_settings,
+                                                        PLUMA_SETTINGS_SIDE_PANEL_SIZE);
+    window->priv->bottom_panel_size = g_settings_get_int (window->priv->editor_settings,
+                                                          PLUMA_SETTINGS_BOTTOM_PANEL_SIZE);
+
+    g_signal_connect_after (window->priv->hpaned,
+                            "map",
+                            G_CALLBACK (hpaned_restore_position),
+                            window);
+    g_signal_connect_after (window->priv->vpaned,
+                            "map",
+                            G_CALLBACK (vpaned_restore_position),
+                            window);
+
+    gtk_widget_show (window->priv->hpaned);
+    gtk_widget_show (window->priv->vpaned);
+
+    /* Drag and drop support, set targets to NULL because we add the
+       default uri_targets below */
+    gtk_drag_dest_set (GTK_WIDGET (window),
+                       GTK_DEST_DEFAULT_MOTION |
+                       GTK_DEST_DEFAULT_HIGHLIGHT |
+                       GTK_DEST_DEFAULT_DROP,
+                       NULL,
+                       0,
+                       GDK_ACTION_COPY);
+
+    /* Add uri targets */
+    tl = gtk_drag_dest_get_target_list (GTK_WIDGET (window));
+
+    if (tl == NULL)
+    {
+        tl = gtk_target_list_new (NULL, 0);
+        gtk_drag_dest_set_target_list (GTK_WIDGET (window), tl);
+        gtk_target_list_unref (tl);
+    }
+
+    gtk_target_list_add_uri_targets (tl, TARGET_URI_LIST);
+
+    /* connect instead of override, so that we can
+     * share the cb code with the view */
+    g_signal_connect (window,
+                      "drag_data_received",
```

Comment:
it's customary to write the name of the signal also on the first line, review this file since there are more matches.